packages/paste-website/src/components/AspectRatio.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+
+interface AspectRatioProps {
+  aspectRatio: string;
+  children: React.ReactElement;
```

Comment:
Type this as `NonNullable<React.ReactNode>` and you can remove the `children == null` check in the validation

Reply:
Done.